erpnext/payroll/doctype/payroll_entry/payroll_entry.py

```diff
@@ -68,7 +70,7 @@ def get_emp_list(self):
 					t1.name = t2.employee
 					and t2.docstatus = 1
 			%s order by t2.from_date desc
-			""" % cond, {"sal_struct": tuple(sal_struct), "from_date": self.end_date}, as_dict=True)
+			""" % cond, {"sal_struct": tuple(sal_struct), "from_date": self.end_date, "payroll_payable_account": self.payroll_payable_account}, as_dict=True)
 			return emp_list
 
 	def fill_employee_details(self):
```

Comment:
improve message for criteria. 
![image](https://user-images.githubusercontent.com/32095923/99657763-9643af80-2a84-11eb-959e-f7a92a2cdabc.png)